dynamicprefixvici.c

```diff
@@ -35,29 +45,80 @@ static void ParseCommandLineArguments(int argc, char** argv, CommandLineArgument
  */
 static void Usage();
 
+/**
+ * @brief Converts a command line argument to an integer
+ * Prints an error message if the argument is invalid and exits
+ * 
+ * @param value 
+ * @param optionname 
+ * @return int 
+ */
+static int parse_option_to_integer(char* value, char* optionname);
+
+/**
+ * @brief Checks whether the string only contains the character specified
+ * Returns true if the string only contains the characters
+ * @param string 
+ * @param characters 
+ * @return true 
+ * @return false 
+ */
+static bool check_only_contains(char* string, char* characters);
```

Comment:
CheckOnlyContains

Reply:
Ik zou deze ook durven onderbrengen in de functie CheckValidityOfArguments, gezien functie maar 1 keer gebruikt wordt, of ook een void en callen vanuit CheckValidityOfArguments

Reply:
Ik vind duidelijker is als dit een aparte functie is.
Op die manier is de logica en code in checkvalidityofarguments duidelijker.
Gezien het een static functie is, kan de compiler, indien gewenst, deze ook inlinen. (kan ook sterk worden aangemoedigd door 'inline' te gebruiken, maar de compiler kan dit vaak zelf beter beoordelen)
Ideaal zou dit een nested functie zijn, maar dat in een GNU-extensie.